<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandWidgetTestWidget</class>
 <widget class="QWidget" name="BandWidgetTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>BandWidth</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="stopButton">
        <property name="font">
         <font>
          <family>Agency FB</family>
         </font>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Agency FB</family>
         </font>
        </property>
        <property name="text">
         <string>Test for 10 times</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_everageBandwidth">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGraphicsView" name="graphicsView"/>
        <widget class="QTableWidget" name="tableWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
